src/Http.ts

```diff
@@ -19,9 +19,46 @@ import * as $T from './Type'
 const ERRORS = {
   BadRequest: 400,
   Unauthorized: 401,
+  PaymentRequired: 402,
   Forbidden: 403,
   NotFound: 404,
+  MethodNotAllowed: 405,
+  NotAcceptable: 406,
+  ProxyAuthenticationRequired: 407,
+  RequestTimeout: 408,
+  Conflict: 409,
+  Gone: 410,
+  LengthRequired: 411,
+  PreconditionFailed: 412,
+  PayloadTooLarge: 413,
+  RequestURITooLong: 414,
```

Comment:
Pascal case makes acronyms work as normal words:
```suggestion
  RequestUriTooLong: 414,
```